config/locales/de/views/users.yml

```diff
@@ -1,82 +1,34 @@
 ---
 de:
   users:
-    confirmations:
-      new:
-        resend_confirmation: Bestätigungsinstruktionen erneut senden
-        validation_error: 'Aufgrund von Fehlern konnten die Instruktionen nicht gesendet werden:'
     mailer:
       confirmation_instructions:
-        action: Meinen Account bestätigen
-        greeting: Willkommen %{name},
-        p1: 'Vielen Dank für Ihre Registrierung bei CodeHarbor! Sie können Ihre E-Mail über den Link unten bestätigen:'
+        p1: 'Vielen Dank für Ihre Registrierung bei CodeHarbor! Klicken Sie auf den folgenden Link, um Ihren Account zu bestätigen:'
       email_changed:
-        greeting: Sehr geehrte/r %{name},
-        p1: Wir kontaktieren Sie, um Ihnen mitzuteilen, dass ihre E-Mail für CodeHarbor geändert wird zu %{unconfirmed_email}.
-        p2: Wir kontaktieren Sie, um Ihnen mitzuteilen, dass ihre E-Mail für CodeHarbor geändert wurde zu %{email}.
+        p1: Wir möchten Sie informieren, dass Ihre E-Mail-Adresse zu %{unconfirmed_email} geändert wurde.
+        p2: Wir möchten Sie informieren, dass Ihre E-Mail-Adresse zu %{email} geändert wird.
       password_change:
-        greeting: Sehr geehrte/r %{name},
-        p1: Wir kontaktieren Sie, um Ihnen mitzuteilen, dass ihr Passwort für CodeHarbor geändert wurde.
+        p1: Wir möchten Sie darüber informieren, dass Ihr Passwort geändert wurde.
       reset_password_instructions:
-        action: Ändere mein Passwort
-        greeting: Sehr geehrte/r %{name},
-        p1: Jemand hat einen Link angefordert, um Ihr Passwort für CodeHarbor zu ändern. Sie können das mit dem Link unten tun.
-        p2: Wenn Sie das nicht waren, ignorieren Sie bitte diese E-Mail.
-        p3: Ihr Passwort wird nicht geändert, bis Sie über den Link oben ein neues erstellen.
+        p1: Jemand hat einen Link zum Ändern Ihres Passworts angefordert. Klicken Sie auf den folgenden Link, um Ihr Passwort zu ändern.
```

Comment:
I assume that those `p1` and `p2` translations can be replaced by Devise, too.

Reply:
They are slightly different from the Devise translations so I kept them but I can replace them too if you like.